### PR TITLE
Look in proto and protobuf for all build tools

### PR DESCRIPTION
**Pull request by chbatey**
* Proto is the norm for maven/gradle with Java
* Protobuf is the norm for SBT with ScalaPB
* Spread proto files across directores for plugin tests

### CODE REVIEW

**Comment by raboof** [comment]
(travis failure should be #325 so a matter of rebasing)

**Comment by chbatey** [comment]
Thanks, rebased and added the suggested comments to the docs.

File: docs/src/main/paradox/maven.md [hunk 1]
```diff
@@ -31,7 +47,9 @@ which is a relative path to the project basedir.
     <artifactId>akka-grpc-maven-plugin</artifactId>
     <version>${akka.grpc.version}</version>
     <configuration>
-      <protoPath>src/main/protobuf</protoPath>
+      <protoPaths>
```
[the remark this answers is not in the document]
Comment (ktoso) [reply docs/src/main/paradox/maven.md]: This was done, cool

File: docs/src/main/paradox/sbt.md [hunk 2]
```diff
@@ -44,9 +44,15 @@ Available parameters are listed in the [ScalaPB documentation](https://scalapb.g
 
 ## Proto source directory
 
+By default protobuf files are looked for in `src/main/protobuf` and `src/main/proto`
```
Comment (ktoso) [inline docs/src/main/paradox/sbt.md]: naisu

**Comment by ktoso** [comment]
Johanness's comment was applied too, merging

**Pull request merged**